### DISCUSSION TITLE
IMS status: not registered

### DISCUSSION THREAD

**Discussion by tchwpkgorg** [kyujin-cho/pixel-volte-patch#151] [General]
After the last Pixel 7 pro update (on 14 June 2023), IMS status on my phone is: "not registered".

In SIM settings, I can see "4G calling" and "Wi-Fi calling" and have them switched on - and yet, they are not working (no "HD" icon next to the call, calling does not work when airplane mode is on and Wi-Fi is connected).

Dialing ##72786# shows "access denied" message.

This is using 2 SIMs: Orange eSIM in Poland; a physical SIM is using a German carrier o2.

Does anyone have any idea how to fix it? 4G calling and Wi-Fi calling worked fine before the last Pixel update.

**Comment by iAmRuSu** [comment]
@tchwpkgorg Yeah, same thing is happening in Romania also. I think the patch needs an update in order to have HD and WiFi Calling active again.

**Reply by ainsztain** [reply]
I added new APS for the internet and working in Plus Poland. 
I have to choose an option: when wifi not available call by cellular provider. Now vowifi is working, but not in a flight mode. Volte also is working. 5g still not.
Today (17Jun) vowifi started working also in a flight mode. Without my intervention.

**Reply by iAmRuSu** [reply]
Vodafone Romania, not working :(

**Comment by ppekrol** [comment]
Same thing happening for eSIM Orange in Poland after June update for Pixel 7 Pro. Couldn't figure out how to make it work.

**Reply by tchwpkgorg** [reply]
I had to add a new APN with "ims" type, as described here: https://github.com/kyujin-cho/pixel-volte-patch/issues/136#issuecomment-1565598716

Now with Orange Flex eSIM in Poland, 5G works, VoLTE works, WIFI calling works!

**Reply by ppekrol** [reply]
Oh, wow. That did the trick. Many thanks!

**Reply by iAmRuSu** [reply]
Vodafone Romania, not working :(
